profile/src/main/java/com/scottlogic/deg/profile/reader/handlers/CreateConstraintsHandler.java

```diff
@@ -102,11 +102,11 @@ private FieldSpecRelations createRelation(RelationalConstraintDTO dto, Fields fi
             case EQUAL_TO_FIELD:
                 return createEqualToRelation((EqualToFieldConstraintDTO) dto, fields);
             case AFTER_FIELD:
-                return new AfterOffsetRelation(main, other, false, dateTimeDefaults, offsetGranularity, dto.offset);
+                return new AfterOffsetRelation(main, other, dto.offset > 0, dateTimeDefaults, offsetGranularity, dto.offset);
             case AFTER_OR_AT_FIELD:
                 return new AfterOffsetRelation(main, other, true, dateTimeDefaults, offsetGranularity, 0);
             case BEFORE_FIELD:
-                return new BeforeOffsetRelation(main, other, false, dateTimeDefaults, offsetGranularity, dto.offset);
+                return new BeforeOffsetRelation(main, other, dto.offset > 0, dateTimeDefaults, offsetGranularity, dto.offset);
```

Comment:
I've changed this to always be inclusive if an offset is used.  I think this behaviour makes the most sense.  See the tests added in this commit to see them